### ISSUE TITLE
Not working with "SELECT TO SPEAK" accessibility. 

### ISSUE THREAD

**Issue by priyaankparekh** [barteksc/AndroidPdfViewer#883]
"SELECT TO SPEAK"  or "TALK-BACK" Accessibility cannot recognize the contents of the PDF.

**Comment by 1stmetro** [comment]
You cannot select to speak as the pdf is basically being displayed as a bitmap you would need to have access to the individual words to allow this function.

**Comment by priyaankparekh** [comment]
@1stmetro  can you tell me how can i achieve that ? and possibly why isn't it working with talkback ?

**Comment by 1stmetro** [comment]
I believe i have seen posts some time ago where people were looking at this possibility but not been on here for a year or so.

Look at the url link code that may help you in the right direction as the links are clickable which is a start so the text is valid if that makes sense.

**Comment by priyaankparekh** [comment]
@1stmetro tried to many things no success ! 
Anyways Thanks.

**Comment by 1stmetro** [comment]
I believe you would need to look at https://github.com/barteksc/PdfiumAndroid which will help you.

**Comment by Bhuvanaarkala07** [comment]
@priyaankparekh  - Is this solved? Are u able to use accessibility on pdfs?

Thanks in advance
